Resources/Help/analyses/correlation_nl.md

```diff
@@ -33,7 +33,7 @@ De Correlatie Matrix maakt het mogelijk de populatiecorrelatie te schatten en de
 - Geef paarsgewijs weer: Geef een tabel weer waarin op elke rij één paar variabelen staat, en de spreidingsdiagrammen worden individueel voor ieder paar weergegeven. Indien onaangevinkt, dan zijn de resultaten weergegeven als matrix, met de namen van variabelen in zowel de kolommen als de rijen.
 - Rapporteer significantie: Geef de p-waarde weer, overeenkomend met de geobserveerde correlatie.
 - Markeer significante correlaties: Markeer statistisch significante correlaties.
-- Betrouwbaarheidsintervallen: Betrouwbaarheidsintervallen voor de populatiecorrelatie (enkel beschikbaar voor de Pearson correlatie).
+- Betrouwbaarheidsintervallen: Betrouwbaarheidsintervallen voor de populatiecorrelatie (betrouwbaarheidsintervallen voor nonparametrische correlaties zijn gebaseerd op de z-benadering van Hollander, Wolfe & Chicken (2013)).
```

Comment:
Betrouwbaarheidsintervallen: Betrouwbaarheidsintervallen voor de populatiecorrelatie (betrouwbaarheidsintervallen zijn exact voor kleine steekproefgroottes, maar op basis van de z-benadering van Hollander, Wolfe & Chicken (2014), wanneer de steekproefgrootte groter is dan 50 voor Kendall's tau en groter dan 1290 voor Spearman's rho).

Reply:
Ik zie dat de .createNonparametricConfidenceIntervals functie altijd de z-benadering gebruikt, ongeacht sample size...

Reply:
Je hebt gelijk, ik zie dat ook en herken ook gedeeltelijk van de code. Op dit moment is er een verwarring bij mij aangaande wat Simon precies aan het implementeren is. De terminologie is een beetje langs elkaar heen gegaan. Weet jij nog waarom wij niet gewoon NSM3::kendall.ci gebruiken?

Reply:
Ik denk dat het toen makkelijker was om even wat code te schrijven dan om een nieuw pakket toe te voegen, maar misschien is het beter om dat pakket te gebruiken. Is Simon bezig met de correlaties nu?

Reply:
Ja Simon, zie ook de PR: https://github.com/jasp-stats/Regression/pull/1/files Het probleem is, is dat alles langzaam liep, omdat voor elk paar .addConcordances(x, y, i) werd aangeroepen. 

Om het trouwens nog ietsjes sneller te maken dacht ik eraan om concordanceSumsVector misschien sneller te maken, door ipv herhaaldelijk cor aan te roepen ipv herhaaldelijk .addConcordances aan te roepen daarna terug te transformeren als in Vergelijking (1) in van Doorn, J., Ly, A., Marsman, M., & Wagenmakers, E.-J. (2018). Bayesian inference for Kendall’s rank correlation coefficient. The American Statistician, 72(4), 303-308.